sky/engine/BUILD.gn

```diff
@@ -22,6 +22,13 @@ config("config") {
   cflags = [
     "-Wno-inconsistent-missing-override",
   ]
+
+  if (is_fuchsia) {
+    # TODO(abarth): Remove once the kernel support unmapping memory properly.
+    defines = [
+      "MEMORY_TOOL_REPLACES_ALLOCATOR",
```

Comment:
IMHO, we should just rip out partition alloc entirely.  I doubt it's giving us much of any benefit.